Makefile

```diff
@@ -17,11 +17,11 @@ tools: tools.timestamp
 tools.timestamp:
 	@go get -u $(BUILDFLAGS) github.com/golang/lint/golint
 	@go get $(BUILDFLAGS) github.com/vbatts/git-validation
-	@go get -u github.com/LK4D4/vndr
+	@go get -u github.com/rancher/trash
 	@touch tools.timestamp
```

Comment:
Hum… use a `tools.timestamp: Makefile` dependency perhaps so that `trash` is automatically downloaded after pulling a version of the repo with this PR merged?